symbulate/probability_space.py

```diff
@@ -138,6 +138,10 @@ def __init__(self, box, size=None, replace=True, probs=None, order_matters=True)
         self.order_matters = order_matters
 
     def draw(self):
+        """A function, that takes no arguments, explaining how to draw one 
```

Comment:
This docstring is not helpful to the user. A user will want to know what .draw() does for a BoxModel. From their perspective, what they want to know is what happens when they call .draw() for a BoxModel. They won't be implementing the .draw() on their own.